### ISSUE TITLE
Has anyone figured out how to make this theme compatible with PHP 8.0?

### ISSUE THREAD

**Issue by dylan-glover** [ThemeFuse/Unyson#4302]
I have a site that is using PHP 7.4 - WPVersion 6.1.1 | Unyson 2.7.28 and the host is forcing everyone to PHP 8.0 - which causes fatal errors.

Has anyone figured out how to make this theme compatible with PHP 8.0?

**Comment by DanSPeterson** [comment]
Same problem here. Would be good if we knew whether this is going to be fixed an updated or not?

**Comment by tazcrzy** [comment]
The hosting company that I use is forcing us to start using PHP 8.0 or they will start charging a fee.  Anyone figured this out yet?

**Comment by dylan-glover** [comment]
> The hosting company that I use is forcing us to start using PHP 8.0 or they will start charging a fee. Anyone figured this out yet?

Yeah I figured it out. I had to debug the theme by looking at the error logs and by turning debug log on. I basically had to convert all old PHP 7.4 syntax, specifically the variables, if I remember correctly, to PHP 8 syntax, then it'll start working.
